### PR TITLE
Push images to Harbor

### PR DESCRIPTION
**Pull request by i7tsov**

### CODE REVIEW

**Comment by i7tsov** [comment]
@tommyongib please open devops ticket to add this repo to https://github.com/orgs/Infoblox-CTO/teams/atlas-threatinsight team with write permission.

**Comment by i7tsov** [comment]
@tommyongib please review.

**Comment by i7tsov** [comment]
Please do not merge yet. I haven't identified Jenkins job that can push image for this repo.

File: Makefile [hunk 1]
```diff
@@ -1,2 +1,30 @@
-include Makefile.vars
-include Makefile.common
+RELEASE_VERSION ?=$(shell grep version colo/charts/farsight-collector-colo/Chart.yaml | awk '{print $$2}')
+
+IMAGE_NAME_COLO ?= farsight.collector.colo
+IMAGE_REGISTRY  ?= infobloxopen
```
Comment (tgjonestx) [inline Makefile]: Should be `infoblox`

File: Jenkinsfile [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  post {
+    success {
+      dir("${WORKSPACE}/${DIRECTORY}") {
+      script {
+          def imageList = ''
+          if (!isPrBuild()) {
+              imageList = sh(script: 'make list-of-images', returnStdout: true)
+          }
+      }
+      finalizeBuild(imageList)
```
Comment (tgjonestx) [inline Jenkinsfile:31]: Remember that you need to run the harbor-repo-init job in Jenkins before the the first image can be pushed

**Review by tgjonestx — CHANGES_REQUESTED** [review]

File: Jenkinsfile (hunk 2 above)
Comment (i7tsov) [reply Jenkinsfile:31]: Yeah, I've already created one in infobloxopen. Will create in infoblox now.

File: Makefile (hunk 1 above)
Comment (i7tsov) [reply Makefile]: Changed to `infoblox`.

**Review by tgjonestx — APPROVED** [review]

**Review by tommyongib — APPROVED** [review]

**Pull request merged**